### PR TITLE
Feat: add bridged udt for account subfield query

### PR DESCRIPTION
**Pull request by Naupio**
link issue:
https://github.com/Magickbase/godwoken_explorer/issues/788

### CODE REVIEW

**Comment by Keith-CY** [comment]
Tests are missing

**Comment by Naupio** [comment]
depend on this pr https://github.com/Magickbase/godwoken_explorer/pull/791

> Tests are missing

**Comment by Keith-CY** [comment]
> depend on this pr #791
> 
> > Tests are missing

Got it

**Review by zmcNotafraid — APPROVED** [review]

**Comment by Naupio** [comment]
testing finished

File: docs/schema.graphql [hunk 1]
```diff
@@ -742,6 +743,47 @@ type RootQueryType {
...
+        type
+      }
+    }
+  }
+  
+  {
+    "data": {
+      "account": {
+        "bridged_udt": {
+          "bridge_account_id": 36050,
```
Comment (Keith-CY) [inline docs/schema.graphql:773]: It seems to be the `native udt id`. And why is `bridge account id` a number while `id` is a string

**Review by Keith-CY — APPROVED** [review]

File: docs/schema.graphql (hunk 1 above)
Comment (Keith-CY) [reply docs/schema.graphql:773]: I misunderstood the updated field

**Pull request merged**